app/controllers/registration_controller.rb

```diff
@@ -148,6 +117,56 @@ def validate_show_registration
       @current_user.to_s == @user_id.to_s || UserApi.can_administer?(@current_user, @competition_id)
   end
 
+  def bulk_update
+    updated_registrations = {}
+    update_requests = params[:requests]
+    update_requests.each do |update|
+      updated_registrations[update['user_id']] = process_update(update)
+    end
+
+    render json: updated_registrations
```

Comment:
I would like to return this like { status: ok, updated_registrations: updated_registrations }

Reply:
Fixed